elements/urth-core-behaviors/jupyter-widget-behavior.html

```diff
@@ -136,19 +137,57 @@
             },
 
             /**
-             * Sends a message to the kernel side proxy of this element.
+             * Sends a message to the kernel side proxy of this element. A promise is return to notify when the message
+             * had been processed by the kernel (i.e. status back to idle). It is not intended to get data, rather meant
+             * to allow clients to throttle messages.
              *
              * @method send
              * @param {Object} msg A json message.
+             * @param {Number} timeout If specified, if timeout expired before status idle is received, promised is rejected
```

Comment:
Expected the description of `timeout` to say something like `The time in (seconds/milliseconds?) to wait for a response before rejecting the promise. Default is ....`